pkgs/top-level/python-packages.nix

```diff
@@ -304,6 +308,8 @@ in {
 
   rhpl = if !isPy3k then callPackage ../development/python-modules/rhpl {} else throw "rhpl not supported for interpreter ${python.executable}";
 
+  salmon = callPackage ../development/python-modules/salmon { };
```

Comment:
This is an application and should therefore be called directly from `all-packages.nix`, and not via `python-packages.nix`.

Reply:
Hmm.. I think I disagree on this. Salmon is to email what Django is to websites. So, I think it should be a package instead of an application. In order to use Salmon, one must write Python modules to define configuration and handlers and apps. Salmon isn't an application that you just launch and it would work on its own. I think we should support both Python 2 and Python 3. I don't see any reason to make Salmon an app, is there some benefit in doing that?

Reply:
Alright, makes sense!